### PR TITLE
Add styles

### PR DESCRIPTION
**Pull request by JuliCarracedo**
This PR by @JuliCarracedo introduces the required styling, based on the given model.
Some strange interaction in the form fields is avoiding them to be aligned with the book containers on top... In 2 hs of work, I wasn't able to find the source of the issue.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,74 @@
...
+
+.icon {
+  fill: #0290ff;
+}
 
-  to {
-    transform: rotate(360deg);
-  }
+.list {
+  width: 50vw;
```
Comment (rukundoeric) [inline src/App.css]: - [ ] <b>REQUIRED:</b> There are still few things to fix to make the app look more like the provided mockup. Kindly fix them.

| Original      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | scrnli_10_1_2021_6-03-31 PM |
  - In the toolbar, the right and left margins must be exactly the same
  - when the title is long, It shouldn't push other elements out of bound.
  - The height of form elements is way too big compared to the height of those elements in the mockup
  - There must be a line separating book list and form

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @JuliCarracedo , 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.css [hunk 2]
```diff
@@ -10,5 +22,182 @@
...
+  opacity: 0.5;
+  font-family: 'Montserrat', serif;
+  font-size: 0.875rem;
+  color: #121212;
+}
+
+.line {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 1.125rem 3.688rem 0 4.938rem;
```
Comment (henatan99) [inline src/index.css]: - [x] I think too much margin in this line is pushing the right element out o the viewport. I suggest you use relative units preferably `%` for the width of the `list-item` children elements. The issue of overflowing elements has been mentioned by the previous reviewer. :pray: 

image

File: src/index.css [hunk 3]
```diff
@@ -10,5 +22,182 @@
...
+
+.sep {
+  border-left-style: solid;
+  border-left-color: #e8e8e8;
+  border-left-width: 1px;
+}
+
+.left {
+  width: 25rem;
+}
```
Comment (henatan99) [inline src/index.css]: - [x] As pointed out by the reviewer, there are still overflowing elements in this class element. Kindly fix them. :pray: 

image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @JuliCarracedo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @JuliCarracedo, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**